bindings/java/nz/mega/sdk/MegaGlobalListenerInterface.java

```diff
@@ -3,9 +3,44 @@
 import java.util.ArrayList;
 
 public interface MegaGlobalListenerInterface {
```

Comment:
No documentation on the interface?

It's common to document the interfaces, and then use on the implementation (e. g. the one that `implements` or `extends`) a `@see` documentation:
http://www.tutorialspoint.com/java/java_documentation.htm